### ISSUE TITLE
Add support for "EdgeInsetsDirectional" in the "padding" extension method!

### ISSUE THREAD

**Issue by devmuaz** [ReinBentdal/styled_widget#59]
Now, despise of the beauty of this package ❤️ ... however, the current ```padding()``` method does not support the directional edge insets like **Flutter** does!
**Flutter has this:**
```dart
@override
Widget build(BuildContext context) {
  return const Scaffold(
    body: Center(
      child: Padding(
        padding: EdgeInsetsDirectional.only(start: 10, end: 10),
        child: Text('View'),
      ),
    ),
  );
}
```
Nowadays, most of apps have localization that could require RTL including: **Arabic**, **Persian**, etc..

I hope you could soon fix it, or I could make a PR for change suggestion.

**Comment by ReinBentdal** [comment]
I am glad you like the package and i would appreciate a pull request! Your contribution with issues and potential PR helps on improving the package

**Comment by devmuaz** [comment]
I'm gonna be closing this issue since we've solved it with this PR #60
